chat_box/tempCodeRunnerFile.py

```
@@ -1 +1 @@
-filter["calories"] = {"$lt": 350}
\ No newline at end of file
+filter=filters
```

Ruff (0.13.3)

Undefined name `filters`

(F821)